### ISSUE TITLE
Color contrast does not meet minimum requirements: collections index page

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5339]
Sites identified: MCO, Archivo
Page(s) identified:

- Collections index page (current collection name in breadcrumb, unit name on collections index page display)

WCAG compliance level: AA

Description:

There is insufficient color contrast between the text and background on this page to meet the minimum contrast requirement.

Text and background colors that conflict or have too little contrast between each other can cause readability issues for users who are color blind or have low vision (vision loss equivalent to approx. 20/40 vision).

To meet the WCAG contrast requirement, you should have a contrast ratio of 4.5:1 for normal text, and 3:1 for large text (18pt and above, or 14pt bold).

See more details in [SiteImprove example](https://my2.siteimprove.com/Inspector/1167098/A11Y/Page?pageId=37958551507&impmd=DB21396D67119C3E30B314CCF5B34C39&conf=a+aa+aria+si&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r69/failed/WUcxQXVyNkU=;ODQ3YjhhNjA=)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbE5nIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--d6c5021bad1a901c756b73101cbbe93b46429d24/image.png)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbGxnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--00e0778b1f77ff6fe261397ac33155f9d0dca0cb/image.png)

# Done Looks Like
- [ ] Collection name text in breadcrumb changed to Black
- [ ] Unit name text changed from light green to Black (or another color from our styling with a stronger color for contrast)

**Comment by elynema** [comment]
Color utilized for unit name is #477076, which passes level AA in the https://webaim.org/resources/contrastchecker/ (note that this color fails at level AAA compliance, unless you make it bold).

Background color for the breadcrumb box is #EAECF4, and making the collection name in the breadcrumb black passes at level AA and AAA of compliance.
